docs/release-notes/discontinue-k80.rst

```diff
@@ -0,0 +1,5 @@
+:orphan:
+
+**Breaking Changes**
+
+-  Cluster: K80 GPUs are no longer supported.
```

Comment:
Do we still support K80s for Pytorch 10 images? Should we add a note for users to use an older image if they have physical k80s?